public/app/features/browse-dashboards/BrowseDashboardsPage.tsx

```diff
@@ -81,6 +83,7 @@ const BrowseDashboardsPage = memo(() => {
   const { data: rootFolder } = useGetFolderQuery('general');
   let folder = folderDTO ? folderDTO : rootFolder;
   const { canEditFolders, canEditDashboards, canCreateDashboards, canCreateFolders } = getFolderPermissions(folder);
+  const hasAdminRights = contextSrv.user.orgRole === OrgRole.Admin;
```

Comment:
Should we check for Grafana admin as well? A user can technically be a Grafana admin and not an org admin.

Reply:
Smth like [this](https://github.com/grafana/grafana/blob/dd7f45011da05f58ccb51d67049199c3b6f7ce9a/public/app/features/alerting/unified/utils/misc.ts#L216), for example.

Reply:
@Clarity-89 In the BE the feature is set up to be available like this: `c.SignedInUser.GetOrgRole() == org.RoleAdmin` (see [here](https://github.com/grafana/grafana/blob/9144e3b44a2327b9ce30c75c2ec258264f510585/pkg/services/navtree/navtreeimpl/navtree.go#L390)). If we want the feature to be available for Grafana admins I assume we need to change the condition in the BE as well. 
What's the actual difference between Grafana admin and org admin?

Reply:
Looks like I've missed that one myself 😅  The condition should be `if s.features.IsEnabled(c.Req.Context(), featuremgmt.FlagDashboardRestoreUI) && (c.SignedInUser.GetOrgRole() == org.RoleAdmin || c.IsGrafanaAdmin) {`. 

Grafana Admin is an administrator for the whole server, I think this is more relevant in the on-prem case. There's some info about this in the [docs](https://grafana.com/docs/grafana/latest/administration/user-management/server-user-management/assign-remove-server-admin-privileges/).